app/controllers/business_support_controller.rb

```diff
@@ -0,0 +1,34 @@
+require "slimmer/headers"
+
+class BusinessSupportController < ApplicationController
+  before_filter -> { set_expiry unless viewing_draft_content? }
+  before_filter :redirect_if_api_request
```

Comment:
I think in previous pull requests we decided to redirect before setting the expiry.